test/e2e/cypress/e2e/sap_systems_overview.cy.js

```diff
@@ -198,10 +212,16 @@ context('SAP Systems Overview', () => {
 
   describe('SAP Systems Tagging', () => {
     before(() => {
-      cy.removeTagsFromView();
+      cy.apiLogin().then(() => {
+        cy.removeTagsFromView();
```

Comment:
Inside `cy.removeTagsFromView()` there is already an `apiLogin()`. Is this intended?

Reply:
Nope, we can remove it good catch!